components/compliance-service/ingest/events/compliance/helper_methods.go

```diff
@@ -116,13 +117,37 @@ func ReportProfilesFromInSpecProfiles(profiles []*inspec.Profile, profilesSums [
 				}
 			}
 
+			refs := make([]relaxting.ESInSpecReportControlRefs, 0)
```

Comment:
If we think the `continue` case is going to be rare, you can do:

```suggestion
			refs := make([]relaxting.ESInSpecReportControlRefs, 0, len(control.Refs))
```

Reply:
i honestly have no idea if the continue case will be rare or not, cause there's no real type checking on the inspec profiles. someone could put completely diff data in that block on the profile, and we would get that on the report :/